creator-node/test/tracks.js

```diff
@@ -445,45 +448,31 @@ describe('test /track_content with actual ipfsClient', function () {
     let storagePath = config.get('storagePath')
     storagePath = storagePath.slice(0, 1) === '/' ? '.' + storagePath : storagePath
 
-    // check if track UUID dir exists
+    // Wait before checking disk state since disk pruning action is made as a non-blocking call and may
+    //  complete after /track_content route returns
+    await utils.timeout(2000)
```

Comment:
this will 100% lead to non deterministic tests. is there a way we can do this deterministically? perhaaps a query. param to make it blocking

Reply:
per discussion, removing the check for artifact pruning in this test as it is designed only for upload, and going to add a separate test to cover just upload artifact pruning

Reply:
done